README.md

```diff
@@ -1,2 +1,65 @@
 # Hello-Microverse
 Apply GitHub flow best practices and provide a good documentation
+
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
```

Comment:
[ ] Great job using the microverse Microverse [README template](https://github.com/microverseinc/readme-template#licenseMicroverse). But there are some missing sections, kindly add them.🤓` (for example the  Built With, set up, and Acknowledgments sections ...)`